### DISCUSSION TITLE
Suggestion to help navigating in the Artist/Albums/Tracks panels

### DISCUSSION THREAD

**Discussion by Waggers58** [music-assistant/support#701] [Feature requests and ideas]
My music collection consists of 2013 albums assigned to 647 (album) artists with a total of 3805 contributing artists.  I am never going to scroll down that lot to find what I want.  The search option only works when you know what you're looking for.  

My suggestion is to copy the alphabet search from Sumsung Music (on my phone).  It shows all the letters of the alphabet down the side and clicking one takes you to the first artist, album or track that starts with that letter and then you can scroll up or down from there.

Anything to speed up scrolling would improve MA.

**Comment by marcelveldt** [comment]
Are you on the latest beta? It contains huge improvements to handle large collections.
Most important the fact that you can filter the "library items" so you can have a large collection but only mark a portion of it as favorites and use that in the filters.

**Comment by Waggers58** [comment]
Hi Marcel

Most people are looking for the integration of extra devices and services but now you had shown me how to mount a network drive, I need no more in that respect.  So, forgive me for concentrating on user access; tell me if you too busy for this just now.  I'm probably ahead of the curve.

Specifying favourites is great but sometimes I want to go "off piste".

Am I right in thinking that currently my options are limited to filtering by favourites or searching through artists or albums for a particular string?  None are great if I just want to browse.

If we could limit the Artists to just album artists that would be a help.  Perhaps we could have a toggle on the Artists panel.

________________________________

**Reply by marcelveldt** [reply]
We're only getting started :-). 
A lot more search and filtering options will be added in further releases now that the basics are covered.
My personal interest is proper suggestions based on tastes and moods and similarities. Your tip for the album artist is great but remember that potentially any artist can be an album artist so not sure if that brings a lot more convenience ?

**Reply by OzGav** [reply]
Actually I’ll second the Album Artist idea. Kodi has that as an option I am pretty sure. It allows you to instantly filter out all the minor artists that appeared on some random various artists album that you aren’t that interested in. There are many possibilities of course….

**Reply by marcelveldt** [reply]
I'll see if I can write a fancy filter for this is one of the next versions.

**Reply by OzGav** [reply]
Album artist filter?…BOOM! Works like a champ..and can be combined with the favourites filter. Thanks @marcelveldt

**Reply by DutchJaFO** [reply]
is there a way to filter by media source ?
I've got both local storage and spotify, but the spotify source appears to have albums/artists that I don't remember selecting anywhere.
The 'heart' filter is too restrictive and the 'album artists' only filter isn't restrictive enough.

I want to use this to check if albums/artists/tracks that I have locally or on Spotify are available in MA itself
